Makefile

```diff
@@ -0,0 +1,6 @@
+install:
+	python onelinerizer/__main__.py .setup.py setup.py
+	pip install .
+
+tests: onelinerizer
```

Comment:
This actually skips the tests unless the timestamp on the `onelinerizer` directory is newer than the timestamp on the `tests` directory. Suggest

```make
.PHONY: install tests

install:
	…

tests:
	…
```

(or just not using `make`, it isn’t really helping us here).